src/components/cc-header-orga/cc-header-orga.js

```diff
@@ -5,6 +5,7 @@ import {
   iconRemixPhoneFill as iconPhone,
```

Comment:
**fix:** in loading mode, the `cc-header-orga` story shows an empty footer, let's investigate in sync when you have time
![image](https://github.com/user-attachments/assets/0fad1df8-78d3-4768-970c-a7a59f3f5af4)